tools/nni_cmd/nnictl_utils.py

```diff
@@ -497,7 +497,8 @@ def experiment_clean(args):
         home = str(Path.home())
         local_dir = nni_config.get_config('experimentConfig').get('logDir')
         if not local_dir:
-            local_dir = os.path.join(home, 'nni', 'experiments', experiment_id)
+            local_dir = os.path.join(home, 'nni', 'experiments')
```

Comment:
default experiment parent dir has been changed to `nni-experiments`  in this release.

Reply:
updated